Gemfile.template

```diff
@@ -2,6 +2,18 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
+
+if Dir.exist?("../logstash-extra")
+  Dir.open("../logstash-extra").each do |entry|
+    next if entry.start_with?('.')
+    path = "../logstash-extra/#{entry}"
+    next unless File.directory?(path)
+    name = entry =~ /^(.+)-logstash$/ ? $1 : entry 
+    puts "Loading development gem: '#{name}' from '#{path}'"
+    gem name, :path => path
+  end
+end
+
```

Comment:
hmmm, I will say that I haven't personally experienced that. If that is the case, I think we should consider that a bug in xpack.

The thing is, for everyone working on xpack they almost always have that line in their Gemfile, so its problematic if that is the case.